src/router/router.ts

```diff
@@ -51,6 +56,16 @@ export function getVueRouter () {
     return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
   }
 
+  /** Returns True if route requires staff role, else False. */
+  function isStaffOnly (route: Route): boolean {
+    return route.matched.some(r => r.meta?.isStaffOnly)
+  }
+
+  /** Returns True if user is staff, else False. */
+  function isStaff (): boolean {
```

Comment:
If this is the only place this is used then I'm OK with it here. Otherwise move it to getter module.

Reply:
As of right now, this is the only place we are verifying the role directly off of the token. 
Why i did it this way, when we mount the app we have the keycloak token in session but we don't have the store(or values) set when attempting the route. This allows us to verify the user is staff before the store is mounted using the `auth-helper.ts`.